### PR TITLE
Output alert applayer v3

### PR DESCRIPTION
**Pull request by catenacyber**
## Tickets

Redmine ticket:
- https://redmine.openinfosecfoundation.org/issues/6501
- https://redmine.openinfosecfoundation.org/issues/6500
- https://redmine.openinfosecfoundation.org/issues/5977

Requires Suricata PR https://github.com/OISF/suricata/pull/9768 https://github.com/OISF/suricata/pull/8961

Rebase of https://github.com/OISF/suricata-verify/pull/1196

### CODE REVIEW

**Comment by victorjulien** [comment]
should be rebased to include https://github.com/OISF/suricata-verify/commit/edef3adf3c9a79bb38fc32f490c55196d94985a1

**Comment by catenacyber** [comment]
Rebased and force-pushed to get Suricata CI greener

File: tests/bug-4903/bug-4903-01/test.yaml [hunk 1]
```diff
@@ -54,12 +54,8 @@ checks:
 - filter:
     count: 1
     match:
-      dest_ip: 192.168.200.1
```
Comment (victorjulien) [inline tests/bug-4903/bug-4903-01/test.yaml:57]: it's unclear why this is removed... did something change in ssh logging?

Comment (catenacyber) [reply tests/bug-4903/bug-4903-01/test.yaml:57]: Not yet in https://github.com/OISF/suricata/pull/9797 v13.1 but in v13.4 there is some change

Pushing a new SV PR without this, and rebasing Suricata PR to reference the new SV PR

**Comment by catenacyber** [comment]
Replaced by https://github.com/OISF/suricata-verify/pull/1482

**Pull request closed without merging**